app/models/linked_account/facebook.rb

```diff
@@ -75,24 +75,35 @@ def self.find_or_create_via_oauth_code(code)
     }
 
     # exchange the code for an access token
-    response = with_https "#{OAUTH_EXCHANGE_URL}?#{params.to_param}" do |uri, http|
-      request           = Net::HTTP::Get.new(uri.to_s)
-      request.add_field "Accept", "application/json"
-      http.request      request
-    end
+    response = HTTParty.get("#{OAUTH_EXCHANGE_URL}?#{params.to_param}")
+
+    
```

Comment:
Try to remove unused code, easier for the next guy :). Can always refer back to older commits if we want to revert mistakes

Reply:
Noted with thanks :)